### PR TITLE
Add about page

### PR DESCRIPTION
**Pull request by YuliyaCher**
I created a text file which contains:
three different types of headings, one at a time;
paragraphs;
one quote;
several line breaks;
several horizontal lines;
at least one tag formatting text strong, em, mark, del, ins, sub, sup;
several special characters and emojis.
link to my GitHub profile;
link to my gmail;

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ghost) [inline about/index.html]: Please make meta tags closed.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <title>Document</title>
```
Comment (ghost) [inline about/index.html]: Please change the title.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+
```
Comment (ghost) [inline about/index.html]: Remove empty line.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <h1>Почему я выбрала Front-end направление.</h1>
+
```
Comment (ghost) [inline about/index.html]: See previous comment.

File: about/index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+<p>Доброго времени суток! Меня зовут Юлия Коленко и я выбрала Front-end разработку, потому, что для меня <b>важно</b>, <br/>как выглядит внешний функционал приложения, с которым контактирует пользователь и интересен сам <i>процесс его создания</i>!</p>
+
```
Comment (ghost) [inline about/index.html]: See previous comment.

File: about/index.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    <li>Умение работать как в комаде так и в одиночку</li>
+    <li>Усидчивость, желание всё доводить до конца</li>
+    <li>Творческие способности</li>
+    <li>Ответственность</li>
+    <li>Быстрое переключение между задачами</li>
+    <li>Умение читать и разбираться в чужом коде</li>
+    <li>Самообучение</li>
+</ol>
+
+<hr style="background: rgb(235, 151, 247);">
```
Comment (ghost) [inline about/index.html]: Please make the tag closed.

File: about/index.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+<ol>
+    <li>Умение работать как в комаде так и в одиночку</li>
+    <li>Усидчивость, желание всё доводить до конца</li>
+    <li>Творческие способности</li>
+    <li>Ответственность</li>
+    <li>Быстрое переключение между задачами</li>
+    <li>Умение читать и разбираться в чужом коде</li>
+    <li>Самообучение</li>
+</ol>
+
```
Comment (ghost) [inline about/index.html]: Remove empty line.

File: about/index.html [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+<hr style="background: rgb(235, 151, 247);">
+
+<p>Однажды Дэн Браун <a href="http://itmydream.com/citati/book/den-braun-cifrovaya-krepost" target="_blank"><mark>сказал</mark></a>: <blockquote>"Возможно всё, на невозможное просто требуется больше времени."</blockquote></p>
+<strong>Когда-то</strong>, и я считала программирование чем-то безумно сложным, <del>невозможным и недоступным</del> для меня.<br/>Сейчас же я понимаю, что нужно просто начать что-то делать, для его понимания, уделять этому <ins>больше времени</ins>, и только тогда <em><b>всё получится!</b></em>
+
```
Comment (ghost) [inline about/index.html]: Remove empty line.

File: about/index.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    <hr style="background: rgb(235, 151, 247);">
```
Comment (ghost) [inline about/index.html]: Seems empty space before <hr> is not necessary. And make it closed.

File: about/index.html [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+<h3>Я не знала, что придумать для использования некоторых других тегов,<br/> поэтому я буду решать <del>сложнейшие</del> уравнения из высшей математики и музыкальной сферы.</h3>
+<p>&#128071;&#128071;&#128071;</p>
+
+&#10112; 1 метр<sup>2</sup> = 100<sup>2</sup> дециметров;
```
Comment (ghost) [inline about/index.html]: Text can't be without an inline tag.

File: about/index.html [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+<br/>
+&#10113;D<sub>7</sub> = b3+m3+m3;
```
Comment (ghost) [inline about/index.html]: See previous comment.

File: about/index.html [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+<hr style="background: rgb(235, 151, 247);">
+<p><a href="mailito: y.chernikava@gmail.com">Вот ссылка</a> на мою почту.</p>
+<p><a href="https://github.com/YuliyaCher" target="_blank">Вот ссылка</a> на мой профиль GitHub.</p>
+<p>&#128204; Прошу понять и простить меня за нехватку полёта фантазии.</p>
+
```
Comment (ghost) [inline about/index.html:34]: Delete empty line.

File: about/index.html [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+    <hr style="background: rgb(235, 151, 247);">
+
+<h3>Я не знала, что придумать для использования некоторых других тегов,<br/> поэтому я буду решать <del>сложнейшие</del> уравнения из высшей математики и музыкальной сферы.</h3>
+<p>&#128071;&#128071;&#128071;</p>
+
+&#10112; 1 метр<sup>2</sup> = 100<sup>2</sup> дециметров;
+<br/>
+&#10113;D<sub>7</sub> = b3+m3+m3;
+
+<hr style="background: rgb(235, 151, 247);">
```
Comment (ghost) [inline about/index.html]: Please make a tag closed.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <title>My First Page</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p><a href="/about/index.html">Об авторе</a></p>
```
Comment (ghost) [inline index.html]: Seems the path to another HTML file is wrong. Try to use "./about/index.html"

File: about/index.html (hunk 1 above)
Comment (YuliyaCher) [reply about/index.html]: Tags closed.

File: about/index.html (hunk 2 above)
Comment (YuliyaCher) [reply about/index.html]: Title renamed.

File: about/index.html (hunk 3 above)
Comment (YuliyaCher) [reply about/index.html]: Empty lines removed.

File: about/index.html (hunk 4 above)
Comment (YuliyaCher) [reply about/index.html]: Empty lines removed.

File: about/index.html (hunk 5 above)
Comment (YuliyaCher) [reply about/index.html]: Empty lines removed.

File: about/index.html (hunk 6 above)
Comment (YuliyaCher) [reply about/index.html]: Tags closed.

File: about/index.html (hunk 7 above)
Comment (YuliyaCher) [reply about/index.html]: Empty lines removed.

File: about/index.html (hunk 9 above)
Comment (YuliyaCher) [reply about/index.html]: Line removed and tag closed.

File: about/index.html (hunk 10 above)
Comment (YuliyaCher) [reply about/index.html]: Yes, I forgot. Wrapped it in a paragraph.

File: about/index.html (hunk 11 above)
Comment (YuliyaCher) [reply about/index.html]: Wrapped it in a paragraph.

File: about/index.html (hunk 12 above)
Comment (YuliyaCher) [reply about/index.html:34]: Line deleted

File: about/index.html (hunk 13 above)
Comment (YuliyaCher) [reply about/index.html]: tag closed

File: index.html (hunk 14 above)
Comment (YuliyaCher) [reply index.html]: Thanks. The path has changed. Should already be working.

File: about/index.html [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+<p>&#128071;&#128071;&#128071;</p>
+<p>&#10112; 1 метр<sup>2</sup> = 100<sup>2</sup> дециметров;</p>
+<br/>
+<p>&#10113;D<sub>7</sub> = b3+m3+m3;</p>
+<hr style="background: rgb(235, 151, 247);"/>
+<p><a href="mailito: y.chernikava@gmail.com">Вот ссылка</a> на мою почту.</p>
+<p><a href="https://github.com/YuliyaCher" target="_blank">Вот ссылка</a> на мой профиль GitHub.</p>
+<p>&#128204; Прошу понять и простить меня за нехватку полёта фантазии.</p>
+</body>
+</html>
```
Comment (ghost) [inline about/index.html:37]: Please follow that rule, code will be prettier and easy to read. https://github.com/hail2u/html-best-practices#other

Comment (YuliyaCher) [reply about/index.html:37]: OK. understood.

**Comment by ghost** [comment]
Good job, don't forget about rule related to prettier code https://github.com/hail2u/html-best-practices#other.

**Pull request merged**